langchain/vectorstores/mongodb_atlas.py

```diff
@@ -21,8 +21,6 @@
 if TYPE_CHECKING:
     from pymongo.collection import Collection
 
-MongoDBDocumentType = TypeVar("MongoDBDocumentType", bound=Dict[str, Any])
```

Comment:
why this change?

Reply:
I got lint error on my local poetry said unbounded type var. I will try to revert back and see if CI passes.